src/app/api/deleteTask/[id]/route.ts

```diff
@@ -0,0 +1,22 @@
+import prisma from "@/lib/prisma";
+
+export async function DELETE(
+  request: Request,
+  { params }: { params: { id: string } }
+) {
+  try {
+    await prisma.task.delete({
+      where: {
+        id: parseInt(params.id, 10),
+      },
+    });
+
+    return new Response(JSON.stringify("this task deleted successfully"), {
+      status: 200,
+    });
+  } catch (error) {
+    return new Response(JSON.stringify({ error: (error as Error).message }), {
+      status: 400,
```

Comment:
**Nit: Generally a status code of 400 means an invalid input, 500 means unidentified server error**

Reply:
done